src/averageAges.js

```diff
@@ -37,7 +51,23 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
```

Comment:
For this function, the wishes are the same as for the previous one.

Reply:
done